server/src/DBTablesUser.cc

```diff
@@ -128,8 +128,8 @@ static const ColumnDef COLUMN_DEF_ACCESS_LIST[] = {
 	NULL,                              // defaultValue
 }, {
 	"host_group_id",                   // columnName
-	SQL_COLUMN_TYPE_BIGUINT,           // type
-	20,                                // columnLength
+	SQL_COLUMN_TYPE_VARCHAR,           // type
```

Comment:
I think we don't always have to support auto migration.

But when we breaks compatibility, we have to notice it users & other developers.
This modification works with old Hatohol DB without warnings and almost works fine.
But it silently contains an incompatibility issue concerned with this column.
It's not good.

How about reject upgrading, show the reason and stop the program at updateDB()?